vehicle/OVMS.V3/components/vehicle_fiatedoblo/src/vehicle_fiatedoblo.cpp

```diff
@@ -0,0 +1,557 @@
+
+    //    { 0x7df, 0, VEHICLE_POLL_TYPE_OBDIIVEHICLE,   0x0A, {  0,  10, 1, 999 }, 0, ISOTP_STD }, // ECU name
+
+    { 0x7e0, 0x7e8, VEHICLE_POLL_TYPE_OBDIIVEHICLE,   0x02, {  0,  10, 15, 25 }, 0, ISOTP_STD }, // VIN //maybe like this or with other POLL type data received... or with poll below
+
+    // does this answer?
+    // { 0x6b4, 0x694, VEHICLE_POLL_TYPE_OBDIIVEHICLE,   0x0a, {  0,  16,   16, 999 }, 0, ISOTP_STD }, // ECU name
+
+    // Battery details
+    { 0x6b4, 0x694, VEHICLE_POLL_TYPE_READDATA,      0xd815, {  0,  5,   1, 999 }, 0, ISOTP_STD }, // Battery voltage
+    { 0x6b4, 0x694, VEHICLE_POLL_TYPE_READDATA,      0xd816, {  0,  5,   1, 999 }, 0, ISOTP_STD }, // Battery current
+    { 0x6b4, 0x694, VEHICLE_POLL_TYPE_READDATA,      0xd86f, {  0,  30,   1, 999 }, 0, ISOTP_STD }, // min cell voltage (mV)
+    { 0x6b4, 0x694, VEHICLE_POLL_TYPE_READDATA,      0xd870, {  0,  30,   1, 999 }, 0, ISOTP_STD }, // max cell voltage (mV)
+    // { 0x6b4, 0x694, VEHICLE_POLL_TYPE_READDATA,      0xd410, {  0,  30,   1, 999 }, 0, ISOTP_STD }, // SOC calibrated (/512 -> %) - not used, as the CAN frame 0x3a8 is also transmitting the value
+    //                                                  0xd440 are single cell details, we don't care about them for now...
+    // { 0x6b4, 0x694, VEHICLE_POLL_TYPE_READDATA,      0xd810, {  0,  30,   1, 999 }, 0, ISOTP_STD }, // SOC, not used, using calibrated one
+    { 0x6b4, 0x694, VEHICLE_POLL_TYPE_READDATA,      0xd865, {  0,  30,   1, 999 }, 0, ISOTP_STD }, // kWh available
+    { 0x6b4, 0x694, VEHICLE_POLL_TYPE_READDATA,      0xd860, {  0,  30,   1, 999 }, 0, ISOTP_STD }, // SOH is (3 byte value - 65536) / 16
+    
+    // VCU Temperature details
+    { 0x6a2, 0x682, VEHICLE_POLL_TYPE_READDATA, 0xd434, {  0,  30,   1,   0 }, 0, ISOTP_STD }, // Ambient temp
+    //    { 0x6a2, 0x682, VEHICLE_POLL_TYPE_READDATA, 0xd49c, {  0,  30,   1,   0 }, 0, ISOTP_STD }, // total milage
+    { 0x6a2, 0x682, VEHICLE_POLL_TYPE_READDATA, 0xd8ef, {  0,  30,   1,   0 }, 0, ISOTP_STD }, // Battery temp
+    { 0x6a2, 0x682, VEHICLE_POLL_TYPE_READDATA, 0xd8cd, {  0,  30,   1,   0 }, 0, ISOTP_STD }, // drive motor stator temperature?
+    { 0x6a2, 0x682, VEHICLE_POLL_TYPE_READDATA, 0xd8ce, {  0,  30,   1,   0 }, 0, ISOTP_STD }, // DC-DC converter temperature or Drive motor coolant inlet temperature?
+    { 0x6a2, 0x682, VEHICLE_POLL_TYPE_READDATA, 0xd8e1, {  0,  30,   1,   0 }, 0, ISOTP_STD }, // on-board charger temperature?
+    { 0x6a2, 0x682, VEHICLE_POLL_TYPE_READDATA, 0xd8f9, {  0,  30,   1,   0 }, 0, ISOTP_STD }, // Traction cicuit coolant temperature * 10
+
+    { 0x6a2, 0x682, VEHICLE_POLL_TYPE_READDATA, 0xd402, {  0,  30,   1,   0 }, 0, ISOTP_STD }, // speed
+   
+    POLL_LIST_END
+  };
+
+uint32_t lastCanFrameTime = 0;
+std::string vin;
```

Comment:
Avoid global variables if possible, use class member variables for these (naming scheme `m_…`).